README.md

```diff
@@ -55,7 +55,7 @@ The development/test database connection details can be found in the .env.exampl
 To start the Oracle database container, run:
```

Comment:
We might wanna change this line to something like "To start the Oracle database container, along with other development service containers, run:"